spec/models/delete_component_event_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe DeleteComponentEvent, type: :model do
+  include CognitoSupport
+
+  context 'on successful component deletion' do
+    it 'component is deleted' do
```

Comment:
not sure this is a correct test name, since it's only about the event, not deletion.. either rename or add the check for the component being deleted